pkg/globalconfig/global_config.go

```diff
@@ -60,13 +60,20 @@ type GlobalConfig struct {
 	ProjectTldGlobal             string                  `yaml:"project_tld"`
 	XdebugIDELocation            string                  `yaml:"xdebug_ide_location"`
 	NoBindMounts                 bool                    `yaml:"no_bind_mounts"`
-	UseTraefik                   bool                    `yaml:"use_traefik"`
+	Router                       string                  `yaml:"router"`
 	WSL2NoWindowsHostsMgt        bool                    `yaml:"wsl2_no_windows_hosts_mgt"`
 	RouterHTTPPort               string                  `yaml:"router_http_port"`
 	RouterHTTPSPort              string                  `yaml:"router_https_port"`
 	ProjectList                  map[string]*ProjectInfo `yaml:"project_info"`
 }
 
+// NewGlobalConfig() returns a minimally initialized GlobalConfig
+func NewGlobalConfig() GlobalConfig {
+	return GlobalConfig{
```

Comment:
Doesn't respect init() func. imho should be better done in a dedicated patch, does not belong to this patch.